docs/src/pages/demos/lists/SimpleList.js

```diff
@@ -24,8 +24,8 @@ function ListItemLink(props) {
 function SimpleList(props) {
   const { classes } = props;
   return (
-    <div className={classes.root}>
-      <List component="nav">
+    <nav className={classes.root} role="navigation">
```

Comment:
Would agree with @oliviertassinari that we don't need this.

If so we should evaluate the every html5 element usage (I think AppBar uses `header`; not sure how this is perceived pre-html5)